part2/countries/src/App.js

```diff
@@ -95,26 +83,34 @@ const App = () => {
   }, [])
 
 
-  const countriesToShow = showCountries
+  const countriesToShow = showList
     ? countries
     : countries
       .filter(country =>
         country.name.toLowerCase()
           .includes(filterCountry.toLowerCase()))
-  // ? filterCountry === ''
-  // : <p>Feel free to search for a country</p>
 
+  console.log(showList)
+  console.log(filterCountry)
 
 
   const handleFilterCountry = (e) => {
     setFilterCountry(e.target.value)
-    setShowCountries(false)
+    setShowList(false)
   }
+
+  const handleButton = (e) => {
+    // es nesaprotu, kas, ko, kāpēc === parentElement.firstChild.data)
+    // atradu, nokopēju, poga strādā pareizi. bet es neatceros, ka FSO par šo būtu mācīts
+    setFilterCountry(e.target.parentElement.firstChild.data)
```

Comment:
Tiksim galā. Kur kopēji?

Reply:
Šis :

https://github.com/marcorichetta/fullstackopen/blob/cd6d68bf04e2fe77436666f207714a65382b94c6/part2/3-countries/App.js#L42